.github/workflows/integration-tests.yaml

```diff
@@ -2,17 +2,24 @@ name: integration test
 on:
   - pull_request
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Adding these parameters, when we push commits to the same PR, the already running Job is forcefully stopped, and new Jobs are started.

So we can reduce CI run times.